pages/tutorials/3.0/audio/sounds.md

```diff
@@ -7,75 +7,106 @@
+The second additional attribute (fourth argument) is the sample rate; it defines how many samples must be played per second in order to reconstruct the original sound.
 
-Now that the audio data is loaded, we can play it with a [`sf::Sound`](https://www.sfml-dev.org/documentation/3.0.0/classsf_1_1Sound.php "sf::Sound documentation") instance.
+Now that the audio data is loaded, we can play it with an [`sf::Sound`](https://www.sfml-dev.org/documentation/3.0.0/classsf_1_1Sound.php "sf::Sound documentation") instance.
 
 ```cpp
 sf::SoundBuffer buffer;
 // load something into the sound buffer...
 
-sf::Sound sound;
-sound.setBuffer(buffer);
+sf::Sound sound(buffer);
 sound.play();
 ```
 
-The cool thing is that you can assign the same sound buffer to multiple sounds if you want. You can even play them together without any issues.
+The cool thing is that you can assign the same sound buffer to multiple sounds if you want.
+You can even play them together without any issues.
 
-Sounds (and music) are played in a separate thread. This means that you are free to do whatever you want after calling `play()` (except destroying the sound or its data, of course), the sound will continue to play until it's finished or explicitly stopped.
+Sounds (and music) are played in a separate thread.
+This means that you are free to do whatever you want after calling `play()` (except destroying the sound or its data, of course), the sound will continue to play until it's finished or explicitly stopped.
 
 ## Playing a music
 
-Unlike [`sf::Sound`](https://www.sfml-dev.org/documentation/3.0.0/classsf_1_1Sound.php "sf::Sound documentation"), [`sf::Music`](https://www.sfml-dev.org/documentation/3.0.0/classsf_1_1Music.php "sf::Music documentation") doesn't pre-load the audio data, instead it streams the data directly from the source. The initialization of music is thus more direct:
+Unlike [`sf::Sound`](https://www.sfml-dev.org/documentation/3.0.0/classsf_1_1Sound.php "sf::Sound documentation"), [`sf::Music`](https://www.sfml-dev.org/documentation/3.0.0/classsf_1_1Music.php "sf::Music documentation") doesn't pre-load the audio data.
+Instead it streams the data directly from the source.
+The initialization of music is thus more direct:
 
 ```cpp
+sf::Music music("music.ogg"); // Throws sf::Exception if an error occurs
+
+// OR
+
 sf::Music music;
 if (!music.openFromFile("music.ogg"))
     return -1; // error
+
 music.play();
 ```
 
-It is important to note that, unlike all other SFML resources, the loading function is named `openFromFile` instead of `loadFromFile`. This is because the music is not really loaded, this function merely opens it. The data is only loaded later, when the music is played. It also helps to keep in mind that the audio file has to remain available as long as it is played.  
+It is important to note that, unlike all other SFML resources, the loading function is named `openFromFile` instead of `loadFromFile`.
```

Comment:
```suggestion
It is important to note that, unlike most other SFML resources, the loading function is named `openFromFile` instead of `loadFromFile`.
```